src/components/LoginModal/LoginModal.js

```diff
@@ -2,112 +2,168 @@
-
-            <label>Email</label>
-            <input
-              className="loginInput"
-              type="email"
-              placeholder="Enter Email"
-              required
-            ></input>
-
-            <label>Password</label>
-            <input
-              className="loginInput"
-              type="password"
-              placeholder="Enter Password"
-            ></input>
-
-            <button type="submit">Login</button>
-            <button onClick={formSwap}>Sign up</button>
-            <div className="rememberMe">
-              <input type="checkbox"></input>
-              <label>Remember me</label>
-            </div>
-            <div>
-              <a href="#!" className="loginMargin">
-                Forgot username
-              </a>
-              <a href="#!" className="loginMargin">
-                Forgot password
-              </a>
-            </div>
-          </form>
-        ) : (
-          <form>
-            <p className="title">Sign Up</p>
-            <label>Full Name</label>
-            <input
-              className="loginInput"
-              type="text"
-              placeholder="Enter Full Name"
-              required
-            ></input>
-            <label>Email</label>
-            <input
-              className="loginInput"
-              type="email"
-              placeholder="Enter Username"
-              required
-            ></input>
-
-            <label>Password</label>
-            <input
-              className="loginInput"
-              type="password"
-              placeholder="Enter Password"
-            ></input>
-
-            <label>Confirm Password</label>
-            <input
-              className="loginInput"
-              type="password"
-              placeholder="Enter Password"
-            ></input>
-
-            <button type="submit">Register</button>
-
-            <div className="rememberMe">
-              <input type="checkbox"></input>
-              <label>Remember me</label>
-            </div>
-            <div>
-              <a href="#!" className="loginMargin" onClick={formSwap}>
-                Go back to Login
-              </a>
-            </div>
-          </form>
-        )}
-      </div>
-    </Modal>
-  );
-};
+class LoginModal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '',
+      email: '',
+      password: '',
+      confirmPassword: '',
+    };
+  }
+
+  onChange = (e) => {
+    this.setState({ [e.target.name]: e.target.value });
+  };
+
+  login = () => {
+    this.props.signInUser(this.state.email, this.state.password);
+    console.log('logging in' + this.state.email, this.state.password);
+  };
+
+  signUp = async () => {
+    console.log('user register clicked');
+    this.props.registerUser(
+      this.state.name,
+      this.state.email,
+      this.state.password
+    );
+
+    await console.log(this.props.user);
```

Comment:
this is a bit trivial to wait for the user to come back to console log it. I would remove the await as I don't really think it has an effect on it too much.

Reply:
oh that was just to test, ill remove it